definitions/checks/foreman_tasks/not_paused.rb

```diff
@@ -5,6 +5,7 @@ class NotPaused < ForemanMaintain::Check
       for_feature :foreman_tasks
       description 'check for paused tasks'
       tags :default
+      after :disk_io, :verify_self_registration
```

Comment:
Why the dependency on this checks? I would recommend to reduce the amount of rules to just those that are justified by the behaviour (such as check on running tasks happening after resuming the paused ones or invalidating the tasks before resuming)